build.sbt

```diff
@@ -882,6 +899,20 @@ lazy val tests =
             Seq((Test / sourceDirectory).value / "scala-2.12+")
           case _ => Nil
         }
+      },
+      // temp hack
```

Comment:
I guess we don't need this comment, it probably wouldn't be jest temporal ;)

Reply:
Yep, was temporary, removed.